examples/ucp_hello_world.c

```diff
@@ -92,18 +94,20 @@ static void set_msg_data_len(struct msg *msg, uint64_t data_len)
 
 static void request_init(void *request)
 {
-    struct ucx_context *ctx = (struct ucx_context *) request;
-    ctx->completed = 0;
+    struct ucx_context *contex = (struct ucx_context *)request;
+    contex->completed = 0;
 }
 
 static void send_handler(void *request, ucs_status_t status, void *ctx)
 {
-    struct ucx_context *context = (struct ucx_context *) ctx;
+    struct ucx_context *context = (struct ucx_context *)request;
+    const char *str             = (const void *)*((void**)ctx);
```

Comment:
why not just pointer to string?

Reply:
since strings are declared with `const` qualifier

Reply:
i guess @brminich ment why const string is not an argument of the func

Reply:
why need to use pointer to pointer? Can just save pointer to the string

Reply:
done